### DISCUSSION TITLE
What credentials is "Join a project" requesting?

### DISCUSSION THREAD

**Discussion by beedell-roke** [BOINC/boinc#5943] [Q&A] [answered]
I've just created an account, and added it to the account manager. However, when I attempt to add a project, I am asked for credentials. Whether this activity is for registration or login isn't stated, so I don't know what credentials to supply (if login) nor what URI I should associate this with in my password manager (is registration):

https://github.com/user-attachments/assets/fb27c59d-749a-48a7-8caa-10625bd17df4

I've read https://github.com/BOINC/boinc/wiki/Choosing-and-joining-projects/b064ef47bf88f6e04c6eb8ed555f38908944ea0f#joining-a-project, but what it states does not match what options I see available to me. Specifically, I see no option to create a new user, which would demonstrate whether I'm looking at a registration or login field. I presume I've misunderstood something?

**Comment by AenBleidd** [comment]
Username and password

**Reply by beedell-roke** [reply]
> https://github.com/BOINC/boinc/discussions/5943#discussioncomment-11520818

@AenBleidd, thank you. However, I presume you didn't read or didn't understand the majority of the OP? If so, undermentioned is what matters:

> When I attempt to add a project, I am asked for credentials. Whether this activity is for registration or login isn't stated, so I don't know what credentials to supply (if login) nor what URI I should associate this with in my password manager (is registration).

What *account*'s credentials is it requesting?

**Reply by AenBleidd** [reply]
When you use Account Manager - you don't need to add separate projects inside the application, just add them on your profile page in the Account Manager (BAM in your case).
If you want to add separate projects - just don't use Account Manager, and provide 'Email', 'UserName' and 'Password' for those projects.

**Reply by beedell-roke** [reply]
> https://github.com/BOINC/boinc/discussions/5943#discussioncomment-11521374

@AenBleidd, in which case, how do I choose which projects to contribute to when using account manager?

**Reply by AenBleidd** [reply]
It's up to you to decide. Select those projects that you like

**Reply by beedell-roke** [reply]
> https://github.com/BOINC/boinc/discussions/5943#discussioncomment-11522252

@AenBleidd, if I do that, they request credentials that I don't have. You definitely seem to think that something which I'm not aware of is obvious, but it's not.

Summarily:

1. I have logged-in with account manager.

   I know these credentials.

5. However, when I want to add a project, I am asked for credentials.
   
   I do not know what credentials this page requests.

**Reply by AenBleidd** [reply]
Where are you trying to add a project: on BAM website or in the Android application?

**Reply by beedell-roke** [reply]
> https://github.com/BOINC/boinc/discussions/5943#discussioncomment-11523078

@AenBleidd, the AOSP application. [The video which I provided](https://private-user-images.githubusercontent.com/42837531/394270250-fb27c59d-749a-48a7-8caa-10625bd17df4.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4NDcwMTAsIm5iZiI6MTczMzg0NjcxMCwicGF0aCI6Ii80MjgzNzUzMS8zOTQyNzAyNTAtZmIyN2M1OWQtNzQ5YS00OGE3LThjYWEtMTA2MjViZDE3ZGY0Lm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEwVDE2MDUxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNhZGJjYjk1MDVjNDk2OWM4YjhlY2VlNmUwM2E2YjI0OWFmMWRmNDAwNmRkNjNlMjFlYzA0ZGI1YjAxNzk5OTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VlUO0DkGkgWWa8iSrOSS5Z5YviM8alT7Jo7Z5abn-z0) in [the OP](https://github.com/BOINC/boinc/discussions/5943#discussion-7649716) surely demonstrates this?

**Reply by AenBleidd** [reply, marked as answer]
1. Go to BAM website: https://www.boincstats.com/bam/signup/
2. Sign-up for projects you like to have
3. In the Android BOINC application click on the **connected** BAM Account Manager and click 'Sync' (or similar, I don't remember the exact title of the menu).
4. **YOU DON'T NEED TO SELECT ANY PROJECTS IN THE APPLICATION WHEN YOU ARE USING ACCOUNT MANAGER!!!**

**Reply by beedell-roke** [reply]
This is seriously unintuitive. The more instructions I read, the less confident I become.

> YOU DON'T NEED TO SELECT ANY PROJECTS IN THE APPLICATION WHEN YOU ARE USING ACCOUNT MANAGER!

1. ~~@AenBleidd, https://lhcathome.cern.ch/lhcathome/join.php, for example, doesn't appear to allow me to register outside of the application. It appears that I must manually add the project. Are some projects incompatible with account manager?~~

1. I've ascertained how to do *this*: one must visit the https://lhcathome.cern.ch/lhcathome/login_form.php page, and *then* can register at https://lhcathome.cern.ch/lhcathome/create_account_form.php?next_url=. What a dreadful UI.

> In the Android BOINC application click on the connected BAM Account Manager and click 'Sync' (or similar, I don't remember the exact title of the menu).

How do I link my disparate project accounts with the account manager?

**Reply by AenBleidd** [reply]
We don't maintain this Account Manager.
Better to ask on their forum.

**Reply by beedell-roke** [reply]
@AenBleidd, apologies. Wasn't aware. Unfortunately, their policy of solely allowing forum posts from users with credits means that I'm in a catch-22.

Regardless, are <em>any</em> projects still active? I ask because I appear to be unable to register at any, whether via my Account Manager: [^1]

[^1]: [`boincstats.com/bam/signup/#tblProjects:~:text=Last%20Attempt`](https://www.boincstats.com/bam/signup/#tblProjects:~:text=Last%20Attempt)

<img width="1924" height="2106" alt="image" src="https://github.com/user-attachments/assets/c18889f6-d7fd-418e-901d-8495ae8b7f0f" />

…or directly, via their websites. Some reject my credentials without stating why; others haven't renewed their Google ReCaptcha certificates, [^2] etcetera.

[^2]: [`denis.usj.es/denisathome/signup.php#rc-anchor-container`](https://denis.usj.es/denisathome/signup.php#rc-anchor-container:~:text=ERROR%20for%20site%20owner%3A%20Invalid%20site%20key)

**Reply by AenBleidd** [reply]
Can't say for all the projects, but the majority of them are active.
We have some volunteers that might be willing to help you with you issues on our BOINC forum: https://boinc.berkeley.edu/forum_index.php

**Reply by beedell-roke** [reply]
@AenBleidd, sincere thanks! However, that forum lacks a registration button:

- #### `firefox-nightly-156.0a1-20260823094236`

  <img width="1275" height="774" alt="Screenshot_20260823_175958" src="https://github.com/user-attachments/assets/f7d0d855-a1d8-4aa0-85c0-1c0db9fa270e" />

- #### Fenix, On AOSP

  <img width="1173" height="2356" alt="Screenshot_20260823-181128" src="https://github.com/user-attachments/assets/7d27f760-a06e-4744-8f5d-850e75d4d9d7" />

Consequently, how am I supposed to submit my information?

**Comment by davidpanderson** [comment]
Oops!  The BOINC site wasn't using the latest software.  Fixed.

**Reply by beedell-roke** [reply]
@davidpanderson, unfortunately, now I can't even access the root path of the domain:

~~~console
PS /> curl https://boinc.berkeley.edu/forum_index.php
curl: (7) Failed to connect to boinc.berkeley.edu port 443 after 162 ms: Could not connect to server
PS /> curl https://boinc.berkeley.edu/
curl: (7) Failed to connect to boinc.berkeley.edu port 443 after 228 ms: Could not connect to server
~~~

Does an issue tracker exist for the website?

**Reply by AenBleidd** [reply]
Works for me.
We don't have issue tracker for the website availability, but we might have a planned maintenance next week.

**Reply by beedell-roke** [reply]
@AenBleidd, thanks. It works for me, now, too. I was forced to utilise my actual e-mail address:

> `aqx02cwi@beedellroke.addy.io` was flagged by `stopforumspam.com`

However, although I observe:

> An email has been sent to `beedell.roke_julian_lockhart+boinc_community.tk8wtg@proton.me`. Visit the link it contains to validate your email address.

[^1] …nothing appears in my inbox, so I am unable to post. Has it actually been sent? I am 1397673.

[^1]: [`boinc.berkeley.edu/forum_post.php`](https://boinc.berkeley.edu/forum_post.php?id=11#:~:text=Please%20validate%20your%20email%20address,please%20validate%20your%20email%20address%2E)
